source/_integrations/signal_messenger.markdown

```diff
@@ -0,0 +1,74 @@
+---
+title: "Signal Messenger"
+description: "Instructions on how to integrate Signal Messenger within Home Assistant."
+logo: signal_messenger.png
+ha_category:
+  - Notifications
+ha_release: 0.102
+---
+
+The `signal_messenger` integration uses the [Signal Messenger REST API](https://github.com/bbernhard/signal-cli-rest-api) to deliver notifications from Home Assistant to your Android or iOs device.
+
+## Setup
+ 
+The requirements are:
+
+- You need to set up the Signal Messenger REST API. 
+- You need a spare phone number to register with the Signal Messenger service. 
+
+
+Please follow those [instructions](https://github.com/bbernhard/signal-cli-rest-api/blob/master/doc/HOMEASSISTANT.md), to set up the Signal Messenger REST API. 
+
+
+## Configuration
+
+To send Signal Messenger notifications with Home Assistant, add the following to your `configuration.yaml` file:
+
+```yaml
+# Example configuration.yaml entry for Signal Messenger 
+notify:
+  - name: signal
+    platform: signal_messenger
+    url: "http://127.0.0.1:8080" # the URL where the Signal Messenger REST API is listening 
+    number: YOUR_PHONE_NUMBER # the sender number
+    recipients: # one or more recipients
+      - RECIPIENT1
+```
+
+{% configuration %}
+name:
+  description: Setting the optional parameter `name` allows multiple notifiers to be created. The notifier will bind to the service `notify.NOTIFIER_NAME`.
+  required: false
+  type: string
+  default: notify
+number:
+  description: The sender number.
+  required: true
+  type: string
+recipients:
+  description: A list of recipients.
+  required: true
+  type: string
+{% endconfiguration %}
+
+
+### Text message
```

Comment:
This one (and the one below) has a level 3 heading? Which means they currently are a subsection of `## Configuration`. But those are automation examples...

Consider adding a level 2 heading just right above this heading.